database/mockDataGen.js

```diff
@@ -0,0 +1,48 @@
+const faker = require('faker');
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
+const writer = csvWriter();
+
+// const generateRandomComments = () => {
+//   randomNumber = faker.random.number({
+//     min: 1,
+//     max: 5
+//   });
+//   let comments = [];
+//   for (let i = 0; i < randomNumber; i++) {
+//     comments.push({
+//       username: faker.internet.userName(),
+//       comment: faker.lorem.sentence(),
+//       profileImgUrl: faker.internet.avatar()
+//     });
+//   }
+//   return JSON.stringify(comments);
+// }
+
+
+// let randomComments = generateRandomComments();
+// console.log(randomComments);
+// console.log(randomComments.slice(0, randomComments.length - 2).split(";,"));
+
+const createCSVdata = () => {
+  console.time('timing seed');
+  writer.pipe(fs.createWriteStream('comments.csv'));
+  for (let i = 0; i < 10000000; i++) {
+    const album_id = i;
+    const commentsObj = JSON.stringify({
+      username: faker.internet.userName(),
+      comment: faker.lorem.sentence(),
+      profileImgUrl: faker.internet.avatar()
+    });
+    writer.write({
+      album_id,
+      commentsObj
```

Comment:
Hey, so are you adding the entire object as a string into the table and then handling it upon querying it in your api calls? Would that not defeat the purpose of using tables?

Reply:
Yeah, that is my intent for now. If I'm using albums as my primary items for handling 10M different endpoints that mean with 1 to 5 comments average to about 30M entries in my database which I'd need to optimize further for.

Reply:
why not `write.write({ album_id, username, comment, profileImageUrl })` instead?